### DISCUSSION TITLE
Setting Hand Slot

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#886] [Help] [answered]
Hello!
How can i set the current selected slot of the player?

So that when he spawns the hand is automaticly at slot 4?

Thanks 👍

**Comment by MrGeoTech** [comment, marked as answer]
I found this. Not sure if it is what you are looking for but it may work. https://javadoc.minestom.net/net/minestom/server/entity/Player.html#setHeldItemSlot(byte)

**Reply by DasLixou** [reply]
Thanks :D
